src/plugins/integrator.rs

```diff
@@ -295,7 +295,11 @@ type ForceComponentsChanged = Or<(
     Changed<ExternalAngularImpulse>,
 )>;
 
-fn clear_forces_and_impulses(mut forces: Query<ForceComponents, ForceComponentsChanged>) {
+
+/// Responsible for clearing forces and impulses on bodies.
+/// 
+/// Public so external crates can order systems after this, runs in [`PhysicsSchedule`], after [`PhysicsStepSet::SpatialQuery`]
```

Comment:
Justifying the publicity here feels unnecessary since it applies to all public systems
```suggestion
/// Responsible for clearing forces and impulses on bodies.
/// 
/// Runs in [`PhysicsSchedule`], after [`PhysicsStepSet::SpatialQuery`].
```